### DISCUSSION TITLE
Browserosaurus 19.1.0 detects Slack and Zoom

### DISCUSSION THREAD

**Discussion by xologram** [will-stone/browserosaurus#576] [General]
Hi folks, just uploaded to 19.1 (from 15, i think) and I noticed that it detects Zoom and Slack and I can't seem to remove them from the list. Is it possible to remove them? Is this a bug? Thanks!

**Comment by will-stone** [comment]
This is by design as it minimises configuration and therefore maintenance. You can drag the apps in to the order you'd like from the Preferences > Apps, then resize the window to hide the ones you don't want to see all the time, this is what I do:
<img width="396" alt="image" src="https://user-images.githubusercontent.com/654103/200538291-0af3a26e-9505-4a58-9a7c-821d6689d73e.png">
<img width="392" alt="image" src="https://user-images.githubusercontent.com/654103/200538323-aa22b4a3-d31b-4c45-89e6-b7a172806025.png">

**Comment by frankvx97** [comment]
Hello @will-stone , I can't add Chrome to the list of browsers, even if I refresh and rescan, Chrome will not appear. How can I fix this?
